mlserver/codecs/numpy.py

```diff
@@ -93,11 +93,11 @@ def encode(cls, name: str, payload: np.ndarray) -> ResponseOutput:
         )
 
     @classmethod
-    def decode(cls, request_input: RequestInput) -> np.ndarray:
-        model_data = _to_ndarray(request_input)
+    def decode(cls, v2_data: Union[RequestInput, ResponseOutput]) -> np.ndarray:
```

Comment:
Interesting! Do you think we should make this the new interface for the `InputCodec` class? I was thinking that we could also include the opposite conversion, although that would probably require splitting the existing `encode` method into `encode_response` and `encode_request`.